tests/train_test.py

```diff
@@ -1,23 +1,29 @@
+from unittest import mock
+
 import pytest
+import tensorflow_datasets as tfds
 from click.testing import CliRunner
+from zookeeper.tf.dataset import TFDSDataset
 
 from larq_zoo.training import basic_experiments
-from tests import dummy_datasets  # noqa
 
 
 @pytest.mark.parametrize("command", list(basic_experiments.cli.commands.keys()))
-def test_cli(command):
+@tfds.testing.mock_data(num_examples=2, data_dir="gs://tfds-data/dataset_info")
+@mock.patch.object(TFDSDataset, "num_examples", return_value=2)
```

Comment:
We patch `TFDSDataset.num_examples` because otherwise training would run for the size of the original dataset.